frontend/src/assets/main.scss

```diff
@@ -124,3 +124,22 @@ td .wrap-block{ width: inherit; }
     text-transform: none;
   }
 }
+
+.p-button {
```

Comment:
to group the selectors, you can change this line to:
`.p-button, .p-inputswitch-slider {`
and you can remove the new style you added in line 134